frontend/src/actions/TracingThunkActions.ts

```diff
@@ -17,7 +17,6 @@ export const TracingThunkActions = {
             if (response.data.data) {
               const trace = transformTraceData(response.data.data, cluster);
               if (trace) {
-                trace.loaded = true;
```

Comment:
Is this no longer needed?

Reply:
No, this is not. It was used when not all the spans info was in the trace, so it was an indicator for the on-hover event. But now, all the info is in the first search call.